Author: Jamie Strandboge <jamie@canonical.com>
Description: Skip borken nodeinfotest to fix FTBFS. This is also disabled in
 the RedHat builds. For details, see:
 http://libvirt.org/git/?p=libvirt.git;a=patch;h=f6770e866547e155d9e6de719d83e62aac6284c6
Forwarded: no

Index: libvirt-0.8.1/tests/nodeinfotest.c
===================================================================
--- libvirt-0.8.1.orig/tests/nodeinfotest.c	2010-06-15 17:02:18.000000000 -0500
+++ libvirt-0.8.1/tests/nodeinfotest.c	2010-06-15 17:05:02.000000000 -0500
@@ -99,6 +99,9 @@
 
     progname = argv[0];
 
+    printf("Skipping %s (BROKEN)\n", progname);
+    return 0;
+
     if (argc > 1) {
         fprintf(stderr, "Usage: %s\n", progname);
         return(EXIT_FAILURE);
